changelogs/fragments/nios-fix-ib_spec.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - nios - added normalizing function to cleanup ``ib_spec`` of extraneous attributes before being merged into ``argument_spec`` (https://github.com/ansible-collections/community.general/pull/1598).
```

How about simplifying the changelog fragment a bit and not labeling it as a bugfix:
```suggestion
minor_changes:
  - nios modules - clean up module argument spec processing (https://github.com/ansible-collections/community.general/pull/1598).
```
and then merging? I'd prefer not to backport this to stable-1, so we're on the safe side.

---

works for me